aspnet/mvc/views/layout.rst

```diff
@@ -0,0 +1,111 @@
+Layout
+=======
+By `Steve Smith`_
+
+Views frequently share visual and programmatic elements. In this article, you'll see how to specify one or more common layouts for your ASP.NET app, and how your views can share directives and run common code before rendering. 
+
+.. contents:: Sections
+	:local:
+	:depth: 1
+	
+`View sample files <https://github.com/aspnet/Docs/tree/master/aspnet/mvc/views/layout/sample>`_
+
+What is a Layout
+----------------
+
+Most web apps have a common layout that provides the user with a consistent experience as they navigate from page to page. The layout typically includes common user interface elements such as the app header, navigation or menu elements, and footer. 
+
+.. image:: layout/_static/page-layout.png
+
+Common HTML structures such as scripts and stylesheets are also frequently used by many pages within an app. All of these shared elements may be defined in a *layout* file, which can then be referenced by any view used within the app. Layouts reduce duplicate code in views, helping them follow the `Don't Repeat Yourself (DRY) principle <http://deviq.com/don-t-repeat-yourself/>`_.
+
+By convention, the default layout for an ASP.NET app is named ``_Layout.cshtml``. The Visual Studio ASP.NET MVC project template includes this layout file in the ``Views/Shared`` folder:
+
+.. image:: layout/_static/web-project-views.png
+
+This layout defines a top level template for views in the app. Apps do not require a layout, and apps can define more than one layout, with different views specifying different layouts.
+
+An example ``_Layout.cshtml``:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/Shared/_Layout.cshtml
+  :language: html
+  :emphasize-lines: 42,66
+
+Specifying a Layout
+-------------------
+
+Razor views have a ``Layout`` property. Individual views specify a layout by defining this property:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/_ViewStart.cshtml
+  :language: html
+  :emphasize-lines: 2
+
+The layout specified can use a full path (example: ``/Views/Shared/_Layout.cshtml``) or a partial name (example: ``_Layout``). When a partial name is provided, the Razor view engine will search for the layout file using its standard discovery process. That is, the controller-associated folder is searched first, followed by the ``Shared`` folder. This discovery process is identical to the one used to discover :doc:`partial views <partial>`.
```

Comment:
`That is, the controller-associated folder is searched first...` => `The controller-associated folder is searched first...`